desktop/release-notes.md

```diff
@@ -24,6 +24,23 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.14.1
+
+2022-11-16
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
```

Comment:
Should we not remove this from 4.14.0?

Reply:
Also the `_includes/desktop-install.html` needs the new checksums in it.

Reply:
just repushed